src/FSharp.Analyzers/FSharp.Analyzers.fsproj

```diff
@@ -6,6 +6,10 @@
         <IsPackable>true</IsPackable>
         <AssemblyName>G-Research.FSharp.Analyzers</AssemblyName>
         <Tailcalls>true</Tailcalls>
+        <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
```

Comment:
This is important to avoid any conflicts with `FSharp.Core`.
Right now this doesn't mix well:
![image](https://github.com/G-Research/fsharp-analyzers/assets/2621499/e779f41f-9e14-4809-b7d7-4f2ce2a94183)

![image](https://github.com/G-Research/fsharp-analyzers/assets/2621499/a1027860-999a-470c-a6c3-77c2a17aee01)